pdfminer/high_level.py

```diff
@@ -16,7 +16,7 @@ def extract_text_to_fp(inf, outfp, output_type='text', codec='utf-8',
                        laparams=None, maxpages=0, page_numbers=None,
                        password="", scale=1.0, rotation=0, layoutmode='normal',
                        output_dir=None, strip_control=False, debug=False,
-                       disable_caching=False, **kwargs):
+                       disable_caching=False, check_extractable=True, **kwargs):
```

Comment:
CI is complaining that this line is too long

Reply:
Do we need **kwargs here? They don't seem to be used